nautiluszim/scraper.py

```diff
@@ -271,9 +281,59 @@ def check_branding_values(self):
                 source=self.about, dest=self.build_dir / "about.html"
             )
 
+    def check_description_length(self):
+        # Checking if the provided input is a file or text
+        if os.path.isfile(os.path.join(os.getcwd(), self.description)):
```

Comment:
we don't support description (or long description) as files (at least for now)

Reply:
Sure, I just thought the additional functionality would be better. Will change accordingly.